jsk_fetch_robot/jsk_fetch_startup/launch/rosbag_play.launch

```diff
@@ -7,6 +7,7 @@
   <arg name="gui" default="false" />
   <arg name="loop_flag" value="--loop" if="$(arg loop)" />
   <arg name="loop_flag" value="" unless="$(arg loop)" />
+  <arg name="play_audio" default="false" />
```

Comment:
how about `audio_play` for arguments?
using both `play_audio` and `audio_play` is quite confusing.

Reply:
OK. I think it is better too.